ext_tables.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+defined('TYPO3') or die();
+
+$GLOBALS['TYPO3_CONF_VARS']['BE']['customPermOptions']['tx_annotate'] = [
```

Comment:
I'm almost 100% certain that this needs to go in ext_localconf.php

Reply:
Really? This only affects the backend, therefore ext_tables.php seems correct.

The documentation also says it's supposed to be in ext_tables.php: https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Backend/CustomPermissions.html